docs/guides/tutorials/getting_started/set_up_data_docs.rst

```diff
@@ -3,65 +3,23 @@
 Set up Data Docs
 ================
 
-:ref:`Data Docs` translate :ref:`Expectations`, :ref:`Validation Results`, and other metadata into clean, human-readable documentation. Automatically compiling your data documentation from your data tests guarantees that your documentation will never go stale---a huge workflow change from the sprawling, chronically-out-of-date data wikis that many data teams otherwise have to maintain.
+:ref:`Data Docs` translate :ref:`Expectations`, :ref:`Validation Results`, and other metadata into clean, human-readable documentation. Automatically compiling your data documentation from your data tests in the form of Data Docs guarantees that your documentation will never go stale.
 
-Go ahead and tell the CLI you want to build Data Docs:
+In the previous steps, when you executed the last cell in the Jupyter notebook, Great Expectations used the Expectation Suite you generated to validate the January data batch. It then compiled those validation results to HTML, and opened a browser window with a Data Docs validation results page:
 
-.. code-block:: bash
+.. figure:: /images/validation_results.png
 
-    Would you like to build Data Docs? [Y/n]: 
-    
-    The following Data Docs sites will be built:
-    
-     - local_site: file:///home/ubuntu/example_project/great_expectations/uncommitted/data_docs/local_site/index.html
-    
-    Would you like to proceed? [Y/n]: 
-    
-    Building Data Docs...
-    
-    Done building Data Docs
-    
-    Would you like to view your new Expectations in Data Docs? This will open a new browser window. [Y/n]: 
+The validation results page is *descriptive*: it describes how a specific batch of data (in this case, the data we profiled) actually looked when validated. The validation results page shows you both your *Expectation* in human-readable form, as well as the *Observed Value* that was found in the data batch you validated, and a *Status* that tells you whether the observed value *passed* or *failed* the validation.
```

Comment:
I want to clarify that Validation Results pages are both prescriptive (like an expectation page) **and** descriptive (which we also display for cases where there are no expectations or only vacuous expectations). The descriptive-only renderer is not covered in this tutorial.


```suggestion
The validation results page is *descriptive*: it describes how a specific batch of data (in this case, the data we profiled) actually looked when validated, by showing the *Observed Value* that was found in the data batch. The validation results page is **also** *prescriptive*: it shows you your normative *Expectation* in human-readable form. Comparing what was expected and what was observed lets the Validation Result provide a *Status* that tells you whether the observed value *passed* or *failed* the validation.
```